rpm/qm.spec

```diff
@@ -90,6 +90,20 @@ automatically isolated from the host. If developers need to further
 isolate there applications from other processes in the QM they should
 use container tools like Podman.
 
+###################
+### qmctl ###
+###################
+
+%package ctl
+Summary:	QM service controller command line tool
+Requires:	%{name} = %{version}-%{release}
+Requires:	python3 >= 3.6
```

Comment:
Why 3.6? I'd aim if possible for some more recent version. At least 3.9 or 3.10 which are still supported nowadays.